docs/CoreConfig.md

```diff
@@ -23,8 +23,8 @@ The list of plugins to load
 
   Keys:
 
-  * port - the port to use (default: 25)
-  * listen\_address - default: 0.0.0.0 (i.e. all addresses)
+  * listen\_host, port - the host and port to listen on (default: ::0 and 25)
```

Comment:
Where has listen_host and listen come from? Where has listen_address gone?

Reply:
@KingNoosh This change is fixing incorrect docs. It is fixing up what is in code with the docs.

The correct config is listen and not listen_address - https://github.com/haraka/Haraka/blob/master/server.js#L258. You seem to have missed the line that adds listen.

listen_host comes from https://github.com/haraka/Haraka/blob/master/server.js#L267. This is used in conjunction with port.